robusta-codegen/src/transformation/imported.rs

```diff
@@ -269,10 +288,22 @@ impl<'ctx> Fold for ImportedMethodTransformer<'ctx> {
                             Pat::Ident(PatIdent { ident, .. }) => ident,
                             _ => panic!("non-ident pat in FnArg")
                         }
-                    },
+                    }
                     _ => panic!("Bug -- please report to library author. Expected env parameter, found receiver")
                 };
 
+                original_signature.inputs
+                    .iter_mut()
+                    .for_each(|i| match i {
+                        FnArg::Typed(t) => match &*t.pat {
+                            Pat::Ident(PatIdent { ident, .. }) if ident == "self" => {}
+                            _ => {
+                               t.attrs.clear();
```

Comment:
Instead of clearing wouldn't it be more robust to just filter out `#[input_type]`?